plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/ExchangeBean.java

```diff
@@ -91,6 +93,13 @@ protected void validatePositivity(BigDecimal... values) {
         }
     }
 
+    protected void validateDate(Instant date, Instant now) {
```

Comment:
you still passing `now` date. Just create now inside this function. No reason to create it everywhere you use this function

Reply:
ok